### PR TITLE
Herm/feat/auth

### PR DESCRIPTION
**Pull request by hermsicle**
Check it out Micky, I think I got something working for the form inputs!

### CODE REVIEW

File: src/components/RegForm/_RegForm.styles.js [hunk 1]
```diff
@@ -28,19 +29,68 @@ export const Title = styled.h1`
...
+	margin-left: 0.5rem;
+	color: #8c8c8c;
+
+	${ ({ focus }) => {
+		if (focus) {
+			return css`
+				animation: moveFocus ease-in-out 0.25s;
+				animation-fill-mode: forwards;
+
+				@keyframes moveFocus {
```
Comment (awwmicky) [inline src/components/RegForm/_RegForm.styles.js:57]: oh, nice approach my dude! using styled-components `css` w/ `@keyframes`
you can technical modularize the keyframe as another file, import it, then apply it

talk to me if you want to find out how

File: src/components/RegForm/index.jsx [hunk 2]
```diff
@@ -38,6 +38,18 @@ RegForm.SubTitle = function RegFormSubTitle ({ children, ...restProps }) {
 	)
 }
 
+RegForm.Frame = function RegFormFrame ({ children, ...restProps }) {
+	return (
```
Comment (awwmicky) [inline src/components/RegForm/index.jsx:42]: what is your thoughts on shrinking all styled components setup:
```js
// before
return (
    <Label { ...restProps }> { children } </Label>
)

// after
return (<Label { ...restProps }> { children } </Label>)
```

**Review by awwmicky — APPROVED** [review]
I'll merge the code & you read the review

File: src/components/RegForm/_RegForm.styles.js (hunk 1 above)
Comment (hermsicle) [reply src/components/RegForm/_RegForm.styles.js:57]: nice, I never thought of it in that way.

File: src/components/RegForm/index.jsx (hunk 2 above)
Comment (hermsicle) [reply src/components/RegForm/index.jsx:42]: It works, I'm just used to doing it that way. I think we can just do a one-liner too without the ( )

**Pull request merged**